profiles/vibrantfix.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/92950980/231708708-e87ee244-335b-4fa6-bc5b-363b24d2e66e.png",
+  "issueId": 1799,
+  "name": "Yogesh",
+  "username": "vibrantfix"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```